editoast/src/models/timetable.rs

```diff
@@ -24,6 +24,19 @@ pub struct Timetable {
     pub id: i64,
 }
 
+impl crate::models::Model for Timetable {
+    type Row = Self;
+    type Changeset = Option<i64>;
+    type Table = editoast_models::tables::timetable::table;
+    type Error = editoast_models::model::Error;
+}
```

Comment:
I do not understand why this piece of code appears in the diff? Didn't we already have an implementation of `Model`? Can't we use `derive(Model)`? (if no, do you think we need a comment to explain why we can't use `derive`?)

Reply:
But `Model` was not implemented on `Timetable` before this PR?

Reply:
`trait Model` wasn't. It wasn't used much except for listing, which is not supported for timetables. But now that we require `Delete: Model` to get `Self::Error`, we need this implementation.

Reply:
Note: the traits are implemented manually because the macro wants to generate a changeset type, which is undefined for objects with just an auto incremented ID.